### PR TITLE
add log export to DMS migration config

### PR DESCRIPTION
**Pull request by taroface**
DOC-9441

### CODE REVIEW

File: src/current/v22.2/aws-dms.md [hunk 1]
```diff
@@ -203,8 +205,8 @@ The `BatchApplyEnabled` setting can improve replication performance and is recom
 
 - For visibility into migration problems:
 
-    - Check the `SQL_EXEC` [logging channel](logging-overview.html#logging-channels) for log messages related to `COPY` statements and the tables you are migrating.
-    - Check the [Amazon CloudWatch logs that you configured](#step-2-2-task-settings) for messages containing `SQL_ERROR`.
+    - Check the [Amazon CloudWatch logs that you enabled](#step-2-2-task-settings) for messages containing `SQL_ERROR`.
+    - Check the CockroachDB logs, especially the `SQL_EXEC` and `DEV` [logging channels]({% link {{ page.version.version }}/logging-overview.md %}#logging-channels), for messages related to `COPY` statements and the tables you are migrating. To access CockroachDB {{ site.data.products.dedicated }} logs, you should have configured log export to Amazon CloudWatch [before beginning the DMS migration](#before-you-begin).
```
Comment (Jeremyyang920) [inline src/current/v22.2/aws-dms.md:209]: If im not mistaken, log export doesnt ship the DEV channel logs which are sometimes important. We only support SESSIONS, OPS, HEALTH, STORAGE, SQL_SCHEMA, USER_ADMIN, PRIVILEGES, SENSITIVE_ACCESS, SQL_EXEC, and SQL_PERF in this manner.

We should also call out that if you don't choose to redact the logs, PII may go to cloudwatch. But if you redact, you might not know the data that caused the issue fully.

**Review by Jeremyyang920 — CHANGES_REQUESTED** [review]

File: src/current/v22.2/aws-dms.md (hunk 1 above)
Comment (taroface) [reply src/current/v22.2/aws-dms.md]: Updated - PTAL!

File: src/current/v22.2/aws-dms.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ Using CockroachDB as a source database within AWS DMS is unsupported.
 
 ## Before you begin
 
-Complete the following items before starting this tutorial:
+Complete the following items before starting the DMS migration:
```
Comment (Jeremyyang920) [inline src/current/v22.2/aws-dms.md:20]: nit: Maybe a bit redundant to say "the DMS Migration". Wondering if "before starting/running DMS" is better as DMS already stands for database migration service.

**Review by Jeremyyang920 — APPROVED** [review]
Changes LGTM! Just the 1 small comment that is up to you.

File: src/current/v22.2/aws-dms.md (hunk 2 above)
Comment (taroface) [reply src/current/v22.2/aws-dms.md:20]: Thanks! I think I'll keep this one, as it's already used throughout the doc and I think is a nice shorthand for "migration that uses DMS". Open to discussion about it later, though.

**Review by Amruta-Ranade — APPROVED** [review]

**Pull request merged**